### PR TITLE
Add dialect issue 101

### PR DESCRIPTION
**Pull request by hramezani**
add dialect option for usePersian #101

### CODE REVIEW

**Comment by Sina7312** [comment]
Finally, thanks a lot...

File: README.md [hunk 1]
```diff
@@ -92,9 +92,11 @@ moment.loadPersian([options])
 
 ### Options
 
-| Param            | Type    | Default | Description                               | Example                                      |
-| ---------------- | ------- | ------- | ----------------------------------------  | -------------------------------------------- |
+| Param            | Type    | Default   | Description                               | Example                                      |
+| ---------------- | ------- | -------   | ----------------------------------------  | -------------------------------------------- |
 | usePersianDigits | Boolean | `false`   | Use persian digits (Use at your own risk) | `moment.loadPersian({usePersianDigits: true})` |
+| ---------------- | ------- | -------   | ----------------------------------------  | -------------------------------------------- |
```
Comment (alitaheri) [inline README.md]: Remove this line. it's not needed between the rows. Only after the header. ( view the file to see the bug)

File: README.md [hunk 2]
```diff
@@ -92,9 +92,11 @@ moment.loadPersian([options])
...
+| dialect          | String  | `persian` | Available values = `persian`, `persian-modern` | `moment.loadPersian({dialect: 'persian-modern'})` |
```
Comment (alitaheri) [inline README.md]: Please change to `dialect *` to indicate a **note** and add a description below the table describing what this option changes.

File: test.js [hunk 3]
```diff
@@ -831,6 +831,21 @@ describe('moment', function() {
...
+      var ol = moment.locale()
+        , m
+      moment.loadPersian({dialect: 'persian-modern'})
+      m = moment('1981-08-21')
+      m.format('D MMMM YYYY').should.be.equal('21 اوت 1981')
+      m.format('jD jMMMM jYYYY').should.be.equal('30 مرداد 1360')
+      m.calendar().should.be.equal('1360/05/30')
+      m.format('LLLL').should.be.equal('جمعه، 30 مرداد 1360 00:00')
+      m.format('llll').should.be.equal('جمعه، 30 مرد 1360 00:00')
+      moment.locale(ol)
```
Comment (alitaheri) [inline test.js]: Please add a test for `weekdaysMin` too 😁.

**Comment by alitaheri** [comment]
Other than the issues I pointed out in the review, everything seems solid.

Thanks a lot 👍 👍

**Comment by hramezani** [comment]
@alitaheri thanks, requested changes are ready

**Comment by alitaheri** [comment]
Beautifully done @hramezani 🎉 🎉

**Pull request merged**